build-data-capturing-gradle-samples/README.md

```diff
@@ -0,0 +1,97 @@
+# Build Data Capturing Gradle Samples
+
+## Overview
+
+This directory contains samples demonstrating various ways to extend and customize a Build Scan with extra data using 
+tags, links, and custom values.
+
+To learn more, see the Gradle Enterprise documentation
+on [Extending build scans](https://docs.gradle.com/enterprise/gradle-plugin/#extending_build_scans). 
+
+### Capture Dependency Resolution
+
+[//]: # (todo figure out what this sample is doing)
+
+### Capture GE Plugin Version
+
+_Demonstrates: Custom values_
+
+This sample demonstrates how to capture the Gradle Enterprise Gradle plugin version as a custom value. This can be used
+to identify projects using a certain version of the plugin within the Build Scan dashboard.
```

Comment:
```suggestion
to identify and filter for projects using a certain version of the plugin within the Build Scan dashboard.
```

@etiennestuder  do we call this screen the Build Scan dashboard? or do we call it something else like Gradle Enterprise dashboard?

Reply:
I applied this suggestion but am unresolving this comment until I hear back from @etiennestuder on the question above ☝️.

Reply:
`...of the plugin on the Gradle Enterprise dashboards`.

Reply:
Updated. Thanks for the guidance.